generator/package.json

```diff
@@ -10,11 +10,11 @@
   "license": "MIT",
   "dependencies": {
     "command-line-args": "^5.0.2",
-    "emoji-datasource": "^4.0.4",
-    "emoji-datasource-apple": "^4.0.4",
-    "emoji-datasource-emojione": "^4.0.4",
-    "emoji-datasource-google": "^4.0.4",
-    "emoji-datasource-twitter": "^4.0.4",
+    "emoji-datasource": "~4.0.4",
+    "emoji-datasource-apple": "~4.0.4",
+    "emoji-datasource-emojione": "~4.0.4",
+    "emoji-datasource-google": "~4.0.4",
+    "emoji-datasource-twitter": "~4.0.4",
```

Comment:
I have set the version of our data set to the last one supporting EmojiOne. To improve backwards compability, I think we should keep it in for 0.6.0 and remove it for the version following that.

Reply:
👍